saleor/core/extensions/dummy_plugin.py

```diff
@@ -0,0 +1,98 @@
+from typing import TYPE_CHECKING, List, Union
+
+from django_countries.fields import Country
+from prices import Money, MoneyRange, TaxedMoney, TaxedMoneyRange
+
+from ..taxes import TaxType, quantize_price
+from .plugin import BasePlugin
+
+if TYPE_CHECKING:
+    from ...checkout.models import Checkout, CheckoutLine
+    from ...product.models import Product
+    from ...account.models import Address
+    from ...order.models import OrderLine, Order
+
+
+class DummyPlugin(BasePlugin):
+    """Dummy plugin class which overwrites all base plugin methods. It should be used
+    when any other plugins are disabled"""
```

Comment:
"any other"

Did you mean when "no plugin are enabled"? Any other sounds like it is enabled when some plugins are disabled.